buildenv/jenkins/variables/defaults.yml

```diff
@@ -395,12 +396,17 @@ x86-64_linux:
   node_labels:
     build:
       all: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
-      8: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-      11: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      8: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
+      11: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
+    docker_image:
+      8: 'eclipseopenj9/jenkins-agent-x86-centos6:13'
+      11: 'eclipseopenj9/jenkins-agent-x86-centos6:13'
```

Comment:
tag 13 or 14?

Reply:
Should be 14 now since that one is built based on head of stream repo content and not my PR. :)  Updated.